tests/usgs_test.py

```diff
@@ -63,16 +74,21 @@ def test_get_usgs_stations():
     stations = usgs.get_usgs_stations()
     assert isinstance(stations, pd.DataFrame)
     assert isinstance(stations, gpd.GeoDataFrame)
-    assert len(stations) > 242000
+    assert 40000 > len(stations) > 35000
```

Comment:
After adding filters based on parameter codes of interest the number of stations in the dataset went way down. Each station might have been listed multiple times, once with each param. And the filter removes entries for parameter codes that exist on a given station, but are not of interest.

https://github.com/SorooshMani-NOAA/searvey/blob/0a3c02e5c9d772b92eceb6392dc73d214a582330/searvey/usgs.py#L128-L130